tests/models/opt/test_modeling_tf_opt.py

```diff
@@ -281,14 +281,14 @@ def test_inference_no_head(self):
 
 
 @require_tf
-@slow
+# @slow
 class TFOPTEmbeddingsTest(unittest.TestCase):
     def setUp(self):
         super().setUp()
         self.path_model = "facebook/opt-350m"
 
     def test_logits(self):
-        model = TFOPTForCausalLM.from_pretrained(self.path_model)
+        model = TFOPTForCausalLM.from_pretrained(self.path_model, from_pt=True)
```

Comment:
Those will need to disappear after the model weights re-upload, but should probably be in a separate PR to merge this one quickly.

Reply:
should also be fixed in the new commit